tiled/server/router.py

```diff
@@ -1215,3 +1219,104 @@ async def delete_revision(
 
     await entry.delete_revision(number)
     return json_or_msgpack(request, None)
+
+
+@router.get("/asset/bytes/{path:path}")
+async def get_asset(
+    request: Request,
+    id: int,
+    relative_path: Optional[Path] = None,
+    entry=SecureEntry(scopes=["read:data"]),  # TODO: Separate scope for assets?
+):
+    if not hasattr(entry, "asset_by_id"):
```

Comment:
Is there any value in also having a top-level server config to enable/disable asset downloading?

Reply:
Is your concern the the fear that we are making it too easy to download huge things and may see unintended consequences?

Reply:
Yes, but I also think it would be inkeeping with Tiled's versatile deployment philosophy. For example I can imagine one facility wanting to hide the raw assets away entirely and use Tiled as a watertight gateway, while another might require raw assets for legacy use cases.

Reply:
That seems like a good idea. And simpler than adding a scope.

Reply:
Added config setting named `expose_raw_assets` (default true) with test in [135dcac](https://github.com/bluesky/tiled/pull/649/commits/135dcac224796191de22b9cae391ca4efd4d6096). If `False`, the two new `/asset/...` routes always return `403 Forbidden` with an explanation.